db/migrate/20181128140935_devise_token_auth_create_users.rb

```diff
@@ -0,0 +1,35 @@
+class DeviseTokenAuthCreateUsers < ActiveRecord::Migration[5.2]
+  def change
+    table = :users
+    change_table(table) do |t|
+      ## Required
+      t.string :provider, null: false, default: "email"
+      t.string :uid, null: false, default: ""
+
+      ## Recoverable
+      t.boolean  :allow_password_change, default: false
+
+      ## Rememberable
+      # t.datetime :remember_created_at unless column_exists?(table, :created_at)
+
+      ## Lockable
+      # t.integer  :failed_attempts, :default => 0, :null => false # Only if lock strategy is :failed_attempts
+      # t.string   :unlock_token # Only if unlock strategy is :email or :both
+      # t.datetime :locked_at
+
+      ## User Info
+      t.string :name unless column_exists?(table, :name)
+      t.string :nickname unless column_exists?(table, :nickname)
+      t.string :image unless column_exists?(table, :image)
+
+      ## Tokens
+      t.json :tokens unless column_exists?(table, :tokens)
```

Comment:
I think we did not have tokens field, remove the `unless`

Reply:
Done!